SofaSimulationCore is missing its include directory when installed

When compiling an external project, doing the following:

**CMakeLists.txt**
```
project(MyProject)
find_package(SofaSimulation)

add_executable(${PROJECT_NAME} main.cpp)
target_link_libraries(${PROJECT_NAME}  SofaSimulationCore)
```

**main.cpp**
`#include <sofa/simulation/MechanicalMatrixVisitor.h>`

Will fail to find the `sofa/simulation/MechanicalMatrixVisitor.h` file, which is supposed to be in the `SofaSimulationCore` target.

Looking closer, we have

 **SofaSimulationTargets.cmake**
```
(...)
# Create imported target SofaSimulationCore
add_library(SofaSimulationCore SHARED IMPORTED)

set_target_properties(SofaSimulationCore PROPERTIES
  INTERFACE_LINK_LIBRARIES "SofaCore"
)

# Create imported target SofaSimulationCommon
add_library(SofaSimulationCommon SHARED IMPORTED)

set_target_properties(SofaSimulationCommon PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include/SofaSimulation"
  INTERFACE_LINK_LIBRARIES "SofaCore;SofaSimulationCore"
)
(...)
```

It seems that the target SofaSimulationCore is missing the line
`INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include/SofaSimulation"`

@guparan I tried to play a little bit with the `sofa/simulation/CMakeLists.txt` to make it work but I failed miserably. You probably have a better idea why this is happening.
______________________________________________________

Suggested labels: <!--- see https://github.com/sofa-framework/sofa/labels -->

Alright, it seems the config.h is also needed. This is what I have by simply putting back SofaSimulationCore in SofaFramework.

**CMakeLists.txt**
```
cmake_minimum_required(VERSION 3.1)
project(mymy)

find_package(SofaFramework)
add_executable(${PROJECT_NAME} main.cpp)
target_link_libraries(${PROJECT_NAME} SofaSimulationCore)
```

**main.cpp**
```
#include <sofa/simulation/MechanicalMatrixVisitor.h>
int main () { return 0;} 
```
**Result**
```
sofa/build/install/include/SofaFramework/sofa/simulation/simulationcore.h:25:10: fatal error: sofa/simulation/config.h: No such file or directory
   25 | #include <sofa/simulation/config.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/SofaKernel/framework/sofa/simulation/CMakeLists.txt
+++ b/SofaKernel/framework/sofa/simulation/CMakeLists.txt
@@ -145,4 +145,4 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread" )
 endif()
 
-sofa_install_targets(SofaSimulation ${PROJECT_NAME} "SofaSimulation/sofa/simulation")
+sofa_install_targets(SofaFramework ${PROJECT_NAME} "SofaFramework/sofa/simulation")

--- a/SofaKernel/framework/sofa/simulation/simulationcore.h
+++ b/SofaKernel/framework/sofa/simulation/simulationcore.h
@@ -22,6 +22,6 @@
 #ifndef SOFA_SIMULATION_CORE_H
 #define SOFA_SIMULATION_CORE_H
 
-#include <sofa/simulation/config.h>
+#include <sofa/config.h>
 
 #endif

